packages/ripple-nuxt-tide/modules/preview/module.js

```diff
@@ -0,0 +1,56 @@
+import { redirect } from './config/paths'
+
+module.exports = function () {
+  const options = this.options.tide
+
+  this.options.proxy = {
+    ...this.options.proxy,
+    '/oauth/token': {
+      target: options.baseUrl
+    }
+  }
+
+  // Using `@nuxtjs/auth-next` instead of `@nuxtjs/auth` due to this bug:
+  // https://github.com/nuxt-community/auth-module/issues/761
+  // Switch to stable when v5 is released.
+  this.addModule(['@nuxtjs/auth-next', {
+    strategies: {
+      drupal: {
+        scheme: 'oauth2',
+        endpoints: {
+          authorization: `${options.baseUrl}oauth/authorize`,
+          token: `/oauth/token`
+        },
+        token: {
+          property: 'access_token',
+          type: 'Bearer',
+          maxAge: 300,
+          name: 'X-OAuth2-Authorization'
+        },
+        refreshToken: {
+          property: 'refresh_token',
+          maxAge: 1209600
+        },
+        responseType: 'code',
+        grantType: 'authorization_code',
+        clientId: options.oauth.clientId,
+        scope: ['editor', 'authenticated']
```

Comment:
@alan-cole If you add option `resetonerror: true` here(https://dev.auth.nuxtjs.org/api/options#resetonerror), user will be automatically logged out if an error happens(For example when token expired). Then we don't need to handle it by ourselves in axios onError event anymore.

Reply:
Should be an easy fix. I'll look at that.

Reply:
@alan-cole My full fix is in https://github.com/dpc-sdp/ripple/pull/796/files

Reply:
Cherry picked your commit into the new branch.